stock_picking_quick/__manifest__.py

```diff
@@ -4,12 +4,13 @@
 
 {
     "name": "Quick Stock Picking",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
     "author": "Akretion, Odoo Community Association (OCA)",
```

Comment:
normally you shouldn't to increment manually

Reply:
You're right, I will let it as is for this one and let the maintainer do a `/ocabot merge nobump`